packages/terra-table/CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 
 ## Unreleased
 
+## 5.2.0 - (November 9, 2023)
```

Comment:
I think we're still doing an alpha release. Can you correct this to `5.1.1-alpha.1`?